java/core/src/main/java/co/worklytics/psoxy/rules/google/PrebuiltSanitizerRules.java

```diff
@@ -146,16 +167,42 @@
         // rules
         //NOTE: reference docs say page elements collection is named 'items', but actual API returns
         // page collection as 'files' or 'revisions'
+        .allowedEndpointRegex("^/drive/v2/files.*")
         .pseudonymization(Rule.builder()
             .relativeUrlRegex("^/drive/v2/files.*")
             .jsonPath("$..emailAddress")
             .build())
         .redaction(Rule.builder()
             .relativeUrlRegex("^/drive/v2/files.*")
+            .jsonPath("$.name") // file display name
+            .jsonPath("$..displayName") //user display name, anywhere (confidentiality)
+            .jsonPath("$..picture") //user picture, anywhere (confidentiality)
+            .jsonPath("$.lastModifyingUserName")
+            .jsonPath("$.items[*].lastModifyingUserName")
+            .jsonPath("$.ownerNames")
+            .jsonPath("$.items[*].ownerNames")
```

Comment:
combine into?
```suggestion
            .jsonPath("$..ownerNames") // owner names, anywhere
```

Reply:
that's just more expensive - it will check everywhere in the hierarchy